### ISSUE TITLE
Pasting comma separated tags

### ISSUE THREAD

**Issue by hammadabbas** [xoxco/jQuery-Tags-Input#178]
Great tags-input control i must say. However if i try to paste a comma separated email addresses the plugin recognises them as one tag initially and only after hitting backspace they are divided into individual tags.

On the demo page http://xoxco.com/projects/code/tagsinput/ try to paste following string and press ','

info@foo.com, contact@bar.com

I made a fix for it, quick and dirty. I am sure you will be able to come up with a better one. Have a look at this jsFiddle:

http://jsfiddle.net/kAWPr/2/

Regards

**Comment by Remo** [comment]
Have the exact same problem, would have been nice to have that in the core, doesn't feel like this is a super rare case. Will try your jsfiddle, thanks!

**Comment by JoshStory** [comment]
I've recently run into a similar issue where pasting doesn't trigger the proper events (in my case, 'onAddTag') since things appear to be bound to keypress.
